src/main/kotlin/services/PersonService.kt

```diff
@@ -41,6 +54,9 @@ class PersonService(
     fun findAll(pageable: Pageable): Page<Person> = repository
         .findAll(pageable)
 
+    fun findAll(pageable: Pageable, active: Boolean): Page<Person> = repository
```

Comment:
Ik zou deze symmetrisch mappen dezelfde naam als in de repository `findAllByActive`